### DISCUSSION TITLE
Please bring back simplelogin.com

### DISCUSSION THREAD

**Discussion by Nastiacute** [simple-login/app#1597] [Q&A]
Please bring back simplelogin.com. If you want to remove simplelogin.fr domain it is not as inconvenient as removing simplelogin.com. The domain simplelogin.com is already added to most whitelists thanks to the efforts of your community and your company which minimizes the risk of registration rejection from this domain. I find your company's decision to remove this particular domain and make it a paid domain incredibly despicable. Why were we able to access this domain when we registered and now not? Why was there no warning? If we had known about your plans, believe me many of us would not have created SinpleLogin accounts. You just tricked your users and put them in a very uncomfortable position.

**Comment by nguyenkims** [comment]
This is duplicated to this discussion https://github.com/simple-login/app/discussions/1589, please continue the discussion there.
